client/network/src/peerset.rs

```diff
@@ -292,9 +267,9 @@ impl Peerset {
 		let (protocol_handles, protocol_controllers): (Vec<ProtocolHandle>, Vec<_>) =
 			controllers.into_iter().unzip();
 
-		let (to_self, from_handle) = tracing_unbounded("mpsc_peerset_messages", 10_000);
+		let (tx, from_handle) = tracing_unbounded("mpsc_peerset_messages", 10_000);
 
-		let handle = PeersetHandle { tx: to_self.clone() };
+		let handle = PeersetHandle { tx };
```

Comment:
I think this change might be the issue, the print "`PeersetHandle` was dropped, terminating `Peerset` is being printed if you enable it.

Reply:
Thanks!!!